bigbluebutton-html5/private/config/settings.yml

```
@@ -481,6 +481,7 @@ public:
     autoSwapLayout: false
     hidePresentation: false
     showParticipantsOnLogin: true
+    layoutControlInActionsMenu: true
```

```suggestion
    layoutControlInActionsMenu: false
```

Hi @Arthurk12 If this was disabled by default I'd be fine with merging in 2.4.
I spoke with @lfzawacki about his https://github.com/bigbluebutton/bigbluebutton/pull/13974 and that points more towards 2.5. (I am mentionining this since PR 13974 contains your PR)

---

Hello @antobinary. Thanks for your notes but we chose to integrate everything together with @lfzawacki and @frankemax's work.